aws.yaml

```diff
@@ -359,6 +359,58 @@ spec:
 ---
 apiVersion: release.giantswarm.io/v1alpha1
 kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  name: v9.3.2
+spec:
+  state: active
```

Comment:
Please deprecate aws 9.2.6 release - it has same k8s major.minor version as 9.3.x.

9.2.6 was deprecated with 9.3.0 release. Once PM was uncovered both affected releases 9.3.1 and 9.3.0 were deprecated, and 9.2.6 re-activated. Now with the fix shipping with 9.3.2, we can deprecate 9.2.6 again.